### ISSUE TITLE
Strange line break

### ISSUE THREAD

**Issue by svnsbck** [EFLS/zetteldeft#133]
Hi! I kind of couldn't find a solution myself. I recently installed Zetteldeft again (Emacs 28.1) and somehow have now a weird line break on the right side. Resizing the screen doesn't change this, also turning on/off visual-line-break doesn't have any effect. This doesn't happen in Deft itself. I guess this should be a bug.

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/8381669/164151945-363159a4-338a-4665-a3c8-9873b46b450c.png">

Any ideas on how to get rid of this?

Any help appreciated.

**Comment by EFLS** [comment]
That's strange. Does it persist after restarting Emacs?

> This doesn't happen in Deft itself

Not sure what you mean, as the screenshot shows the Deft window.

Seems related to: https://github.com/jrblevin/deft/issues/60
I remember having similar issues some time ago, not sure how it disappeared (after an update I think).

**Comment by svnsbck** [comment]
Sorry I wasn't clear. I meant, that the behavior appeared after I installed Zetteldeft, before that Deft was fine. 

I tried also re-starting and out-commenting Zetteldeft and restarting and the result was the same in Deft. 

Weirdly the problem "fixed" itself after I tried some of the suggestions in the issue you linked. Neither of them did "restore" it to the way it was, before, but after yet another restart things were fine again.

Strange, but I consider the problem fixed. Sorry to have bothered you and thanks for your help.
